rad/monadic/issues.rad

```diff
@@ -4,20 +4,37 @@
 ;; Create a remote issue chain. Returns a ref for the chain that you
 ;; can assign to a variable.
 
-(load! "rad/prelude.rad")
+(load! "rad/prelude/test.rad")
+
+(file-module! "rad/prelude/basic.rad")
+(import prelude/basic :unqualified)
+
+(file-module! "rad/prelude/io.rad")
+(import prelude/io :unqualified)
+
+(file-module! "rad/prelude/ref.rad")
+
+(file-module! "rad/prelude/dict.rad")
+(import prelude/dict :unqualified)
+
+(file-module! "rad/prelude/lens.rad")
+(import prelude/lens :unqualified)
+
+(file-module! "rad/prelude/chain.rad")
+(import prelude/chain 'chain)
 
 (def radicle-issues-chain-id
   "The name of this chain."
   "monadic/radicle/issues")
 
-(def issues-chain!
-  (ref (load-chain! "http://radicle.xyz/chains/radicle-issues")))
+(def issues-chain
+  (ref (chain/load-chain! "localhost:8000/chains/radicle-issues")))
```

Comment:
Hm. Maybe parametrize the module by url? @geigerzaehler were just talking about this in relation to the tests.

Reply:
Not sure what's best here so I'll leave it to a later PR.

Reply:
Sounds good.